### PR TITLE
 Update MemoryOp to contain a single byte per value 

### PR DESCRIPTION
**Pull request by CPerezz**
Resolves: #96

### CODE REVIEW

File: bus-mapping/src/evm/opcodes/mload.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@ use crate::{
     operation::{container::OperationContainer, MemoryOp, StackOp, RW},
     Error,
 };
-use core::{convert::TryInto, ops::Deref};
+use core::convert::TryInto;
 
 /// Number of ops that MLOAD adds to the container & busmapping
 const MLOAD_OP_NUM: usize = 3;
```
Comment (han0110) [inline bus-mapping/src/evm/opcodes/mload.rs:11]: Missed update.
```suggestion
const MLOAD_OP_NUM: usize = 34;
```

File: bus-mapping/src/evm/opcodes/push.rs [hunk 2]
```diff
@@ -47,3 +47,106 @@ impl Opcode for Push1 {
         Ok(PUSH1_OP_NUM)
     }
 }
+
+#[cfg(test)]
+mod mload_tests {
```
Comment (han0110) [inline bus-mapping/src/evm/opcodes/push.rs:52]: Missed update.

```suggestion
mod push1_tests {
```

File: bus-mapping/src/evm/opcodes/push.rs [hunk 3]
```diff
@@ -47,3 +47,106 @@ impl Opcode for Push1 {
...
+        evm::{
+            EvmWord, GasCost, GasInfo, Memory, OpcodeId, ProgramCounter, Stack,
+            StackAddress,
+        },
+        BlockConstants, ExecutionTrace,
+    };
+    use pasta_curves::pallas::Scalar;
+
+    #[test]
+    fn mload_opcode_impl() -> Result<(), Error> {
```
Comment (han0110) [inline bus-mapping/src/evm/opcodes/push.rs:64]: Missed update.
```suggestion
    fn push1_opcode_impl() -> Result<(), Error> {
```

**Review by han0110 — COMMENTED** [review]
Some nitpick, others LGTM!

**Review by miha-stopar — APPROVED** [review]
Cool! Thanks, Carlos. I have no comments (I saw you remove memory_values test but this makes total sense as it checks for memory to be byte and now MemoryOp value is u8 and thus always < 256).

**Pull request merged**